### DISCUSSION TITLE
Anilist isn't working in my profile

### DISCUSSION THREAD

**Discussion by Oreki-san** [lowlighter/metrics#676] [Q&A] [answered]
![Screenshot_2021-11-24-12-55-49-65_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/77143046/143193668-c0766072-2a13-4dac-a834-56dd25234475.jpg)
Can you tell me what's the reason

**Comment by spenserblack** [comment, marked as answer]
The plugin defaults to using the same username as GitHub.
https://github.com/lowlighter/metrics/blob/d3b5a9d8c68562bd41c698937d2eac23181bdb95/source/plugins/anilist/metadata.yml#L62-L65

Do you need to set `plugin_anilist_user` to be a different username?

**Reply by lowlighter** [reply]
https://anilist.co/user/Oreki-san returns a 404 so that's probably the reason
